webpack_loader/templatetags/webpack_loader.py

```diff
@@ -1,18 +1,46 @@
-from django import template, VERSION
 from django.conf import settings
+from django.template import Library
 from django.utils.safestring import mark_safe
 
 from .. import utils
 
-register = template.Library()
+register = Library()
+_PROC_DJTEMPLATE = 'django.template.context_processors.request'
+_DJ_TEMPLATEPROCESSOR = 'django.template.backends.django.DjangoTemplates'
+_STARTUP_ERROR = (
+    f'Please make sure that "{_PROC_DJTEMPLATE}" is added '
+    'to your ["OPTIONS"]["context_processors"] list in your '
+    f'settings.TEMPLATES where the BACKEND is "{_DJ_TEMPLATEPROCESSOR}". '
+    'django-webpack-loader needs it and cannot run without it.')
 
 
-@register.simple_tag
-def render_bundle(bundle_name, extension=None, config='DEFAULT', suffix='', attrs='', is_preload=False):
+def _is_request_in_context():
+    for item in settings.TEMPLATES:
+        backend = item.get('BACKEND', {})
+        if backend == _DJ_TEMPLATEPROCESSOR:
+            processors = set(
+                item.get('OPTIONS', {}).get('context_processors', []))
+            if _PROC_DJTEMPLATE not in processors:
+                raise RuntimeError(_STARTUP_ERROR)
+
+
+# Check settings at module import time
+_is_request_in_context()
```

Comment:
@fjsj I don't have much knowledge in django so I'd like your input here.

Reply:
As mentioned in #296 (I think), the request is added to the context as per the default middleware configuration in django. This only fails (halts the startup of the app at the earliest) when somebody removed that middleware for some reason. Should that happen, this will give a descriptive explanation as to why the module halted execution.

Reply:
Got it, thanks for the context.

I think this makes sense, just want to confirm with Flávio because he has more knowledge in Django than me, but I think there are no issues here.